dev-tools/vendor/github.com/cavaliercoder/go-rpm/packagefile.go

```diff
@@ -13,7 +13,7 @@ import (
 	"time"
 )
 
-// A PackageFile is an RPM package definition loaded directly from the pacakge
+// A PackageFile is an RPM package definition loaded directly from the package
```

Comment:
Can you remove this one please? It's in a dependency so it doesn't help to much to fix the typo in our vendored copy.